internal/runtimehandlerhooks/high_performance_hooks.go

```diff
@@ -289,3 +310,63 @@ func setCPUQuota(cpuMountPoint, parentDir string, c *oci.Container, enable bool)
 
 	return nil
 }
+
+func setRPSLoadBalancing(c *oci.Container, sysDevicesDir, onlineCPUsSysFile string, enable bool) error {
+	lspec := c.Spec().Linux
+	if lspec == nil ||
+		lspec.Resources == nil ||
+		lspec.Resources.CPU == nil ||
+		lspec.Resources.CPU.Cpus == "" {
+		return errors.Errorf("find container %s CPUs", c.ID())
```

Comment:
Should the error message be `cannot find...`?

Reply:
I paraphrased https://github.com/cri-o/cri-o/blob/ae6220f779afb6c936d95816540021258ef5a5b2/internal/runtimehandlerhooks/high_performance_hooks.go#L151.
You are right and I will make the change.